<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gamma_0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rho_0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>U_0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>P_0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Gamma_3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Rho_3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>U_3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>P_3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>129</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGamma_0">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRho_0">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxU_0">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxP_0">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>100000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGamma_3">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRho_3">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxU_3">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxP_3">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>100000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>491</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>eps</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxEps">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="value">
     <double>0.000010000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
